CONTRIBUTING.md

```diff
@@ -434,12 +435,49 @@ class UserBio {
 
 
 
+## Backend connectivity
+
+### Local development
+
+Development [GraphQL] API playground is available [here][4].
+
+In order to connect to the development backend [GraphQL] endpoint, you should either use the following `--dart-define`s:
+
+```bash
+--dart-define=SOCAPP_HTTP_URL=https://messenger.soc.stg.t11913.org
+--dart-define=SOCAPP_WS_URL=wss://messenger.soc.stg.t11913.org
+--dart-define=SOCAPP_HTTP_PORT=443
+--dart-define=SOCAPP_WS_PORT=443
+--dart-define=SOCAPP_CONF_REMOTE=false
+```
+
+__Or__ pass the following configuration to `assets/conf.toml`:
+
+```toml
+[conf]
+remote = false
+
+[server.http]
+url = "https://messenger.soc.stg.t11913.org"
+ort = 443
+
+[server.ws]
+url = "wss://messenger.soc.stg.t11913.org"
+port = 443
+```
+
+__Note__, that passing `--dart-define`s to `make e2e` or `flutter drive` commands is __not__ possible. Use the `assets/conf.toml` in order to run E2E tests on the development backend. Be sure not to commit the modified `assets/conf.toml` file.
```

Comment:
@SleepySquash as a separate minor issue, we need to think about some local configs layering here, just for development purpose.